Library/Homebrew/dev-cmd/bump-formula-pr.rb

```diff
@@ -469,7 +469,8 @@ def run
         pr_title = if args.bump_synced.nil?
           "#{formula.name} #{new_formula_version}"
         else
-          "#{Array(args.bump_synced).join(" ")} #{new_formula_version}"
+          bump_synced = Array(args.bump_synced)
+          "#{bump_synced.first(5).join(" ")}#{" ..." if bump_synced.count > 5} #{new_formula_version}"
```

Comment:
- use `maximum_formulae_in_title` or a similar variable
- feels like it might be nicer to do something like:
```ruby
def truncate(str, max_length)
  return str if str.length <= max_length
  str[0...(max_length - 3)] + "..."
end
```

and trim the string to the actual maximum length rather than hoping the number of formula names limits it

Reply:
> I think 72 characters was GitHub's default wrap in web UI but haven't confirmed if this is still the case.

think it's a fair bit more. you can play around pretty easily in a test repo.

Reply:
@cho-m apologies for butchering this PR title 😂. Seems to be a lot more than 72.

Reply:
@cho-m where in the UI are you looking for this. Commits? `brew log`?

Reply:
Just when opening a new PR, anything over 72 characters gets wrapped to body so assuming it is GitHub's recommendation (https://github.com/orgs/community/discussions/12450)

Anyways, seems more a question now long titles impact readability if there isn't a strict hard limit. 
- Qt sync can exceed 550 characters.
- Vulkan is currently around 150 characters. If we add SPIRV, maybe 190 characters.